### ISSUE TITLE
Enhancement: Special case of replacing if statement with if expression

### ISSUE THREAD

**Issue by gotdang** [sourcery-ai/sourcery#81]
### Issue description or question
I work with several jr programmers, and see snippets like this:
```python
    if some_boolean_expression:
        some_var = False
    else:
        some_var = True
```
Sourcery recommends changing this to:
`some_var = False if some_boolean_expression else True`

This is fine, but for the special case of False/True, it would be simpler to write:
`some_var = not some_boolean_expression`

This also applies when the True/False are swapped, by removing the `not`.

### Sourcery Version
0.7.11

Often available in Sourcery settings

### Code editor or IDE name and version

VSCode 1.42.8, running just fine on WSL:Debian

### OS name and version

Windows 10, running WSL:Debian 10.,4

**Comment by brendanator** [comment]
Thanks for the suggestion!

One little point to note is that if `some_expression` is not a boolean in the following:
```python
    if some_expression:
        some_var = True
    else:
        some_var = False
```

then it should be wrapped in `bool` to ensure the same meaning:
```python
some_var = bool(some_expression)
```

**Comment by Hellebore** [comment]
This will be in the next release.

**Comment by Hellebore** [comment]
Released with 0.7.12
